### PR TITLE
add new:the new stack

### LINKED ISSUE #834: [NG] Add The New Stack: Meshery in top 3 service mesh developments in 2020 news article
**Desired Behavior**
Add `The New Stack: Meshery in top 3 service mesh developments in 2020` news article present on the [current site](https://layer5.io/news/meshery-in-top3-service-mesh-developments-in-2020) to layer5ng. Check[ #815](https://github.com/layer5io/layer5/issues/815) for more more details. 

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by lolyparty**
**Description**

This PR fixes #834 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/news/2019-12-12-meshery-in-top3-service-mesh-developments-in-2020/index.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "The New Stack: Meshery in top 3 service mesh developments in 2020"
+subtitle: "A sample subtitle about growing together as a community and building a project that matters"
```
Comment (Tanuj22) [inline src/collections/news/2019-12-12-meshery-in-top3-service-mesh-developments-in-2020/index.mdx:3]: Leave this blank. LIke `subtitle: ""`

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Awesome Work :) . Just one change and we are good to go here.

File: src/collections/news/2019-12-12-meshery-in-top3-service-mesh-developments-in-2020/index.mdx (hunk 1 above)
Comment (lolyparty) [reply src/collections/news/2019-12-12-meshery-in-top3-service-mesh-developments-in-2020/index.mdx]: Done

**Review by Tanuj22 — APPROVED** [review]
Thank you @lolyparty :)

**Pull request merged**